react-toggle-switch-component/src/toggle.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+class Toggle extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      toggle: false,
+      color: '',
+      onOrOff: ''
+    };
+    this.handleToggle = this.handleToggle.bind(this);
+  }
+
+  handleToggle() {
+    this.setState({ toggle: !this.state.toggle });
+  }
+
+  render() {
+    const $body = document.querySelector('body');
```

Comment:
Don't ever query parts of the `document` from inside a react app. Instead of changing the background color of the page, resize the React app's element to cover the page.